backend/modules/extract/domain/events.py

```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from ...data.domain.events import DomainEvent
+
+
+# TODO: discuss what kind of information we want to pass here.
```

Comment:
This comment should be removed.

Regarding question:
We can't put data here, since it can be really huge and at this moment we don't need it.
It's enough for now to put file path as it is.